src/hint/README.md

```diff
@@ -0,0 +1,33 @@
+# Hint Timer
+
+The hint timer is a simple resettable timer.
```

Comment:
I think it's worth also providing a use case here. Also, we'll want to link to this readme from the main one in the root folder (with a short description there as well)

Reply:
I second the request for use case/description documentation. SpringRoll 1 has a HintsPlayer class, but we never used it; we just rolled our own hint/prompt logic per game. Curious to know what the idea of the global hint timer and hint button at the Container level are about.